src/libvmemcache.h

```diff
@@ -90,6 +87,18 @@ enum vmemcache_replacement_policy {
 	VMEMCACHE_REPLACEMENT_NUM
 };
 
+enum vmemcache_statistics {
```

Comment:
drop the last `s` ;) `vmemcache_statistic`

Reply:
Done